packages/web-components/@sfx/sayt/src/sayt.ts

```diff
@@ -194,14 +201,26 @@ export default class Sayt extends LitElement {
         sfx-sayt[hidden] {
           display: none;
         }
+
+        .sfx-sayt-container {
+          display: flex;
+        }
       </style>
-      ${ this.showCloseButton ?
-        html`<button aria-label="Close" @click=${ this.clickCloseSayt }>
-          ${ this.closeText }
-        </button>`
-        : ''
-      }
-      ${ this.hideAutocomplete ? '' : html`<sfx-autocomplete></sfx-autocomplete>` }
+        ${this.showCloseButton
+          ? html`
+              <button aria-label="Close" @click=${this.clickCloseSayt}>
+                ${this.closeText}
+              </button>
+            `
+          : ''}
+        <div class="sfx-sayt-container">
```

Comment:
Is this container necessary?

Reply:
If we have a close button, I think it makes it easier to do flexbox if we have it.